Wikipedia/Code/MWKLanguageLink.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithLanguageCode:(nonnull NSString *)languageCode
 }
 
 - (BOOL)isEqualToLanguageLink:(MWKLanguageLink *)rhs {
-    return WMF_RHS_PROP_EQUAL(languageCode, isEqualToString:) && WMF_RHS_PROP_EQUAL(pageTitleText, isEqualToString:) && WMF_RHS_PROP_EQUAL(name, isEqualToString:) && WMF_RHS_PROP_EQUAL(localizedName, isEqualToString:) && [self.namespaces isEqualToDictionary:rhs.namespaces];
+    return WMF_RHS_PROP_EQUAL(languageCode, isEqualToString:) && WMF_RHS_PROP_EQUAL(pageTitleText, isEqualToString:) && WMF_RHS_PROP_EQUAL(name, isEqualToString:) && WMF_RHS_PROP_EQUAL(localizedName, isEqualToString:) && WMF_RHS_PROP_EQUAL(localizedName, isEqualToString:);
```

Comment:
Did you mean for the addition to be `WMF_RHS_PROP_EQUAL(namespaces, isEqualToDictionary:)`?

Reply:
Yes, yes I did. Egg on my face. Updated now.